data/yam/agama/auto/default_sle.json

```diff
@@ -12,5 +12,13 @@
   "root": {
     "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
     "hashedPassword": true
+  },
+  "scripts": {
+    "posts": [
+      {
+        "name": "enable sshd",
+        "body": "#!/bin/bash \n echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/10-PermitRootLogin.conf \n firewall-cmd --permanent --add-service=ssh \n firewall-cmd --reload \n systemctl enable sshd \n"
```

Comment:
I guess you should able to put each command in a different line for readability.

Reply:
not implemented at the moment, we have to use _jsonnet_ in case, but for few commands the escape `\n` is enough, I think

Reply:
See [docs about unattended/scripts](https://agama-project.github.io/docs/user/unattended/scripts)